samples/base/main.cpp

```diff
@@ -1,5 +1,40 @@
-#include <lug/Config.hpp>
+#if defined(__ANDROID__)
+ #include <android/log.h>
+ #define LOGV(...)  ((void)__android_log_print(ANDROID_LOG_VERBOSE, "lugdunum sample", __VA_ARGS__))
+ #include "android_native_app_glue.h"
+ #include "Lug/System/Exception.hpp"
+#endif
 
-int main() {
+#if defined(__ANDROID__)
+    void android_main(android_app* app)
+#else
+    int main()
+#endif
+{
+#if defined(__ANDROID__)
+    app_dummy();
+    while (1)
+    {
```

Comment:
`{` goes at the end of previous lines. Applies to the `while` and the `if`s below too.